ui/internal-utilities/src/utils/time-series/mockTimeSeriesResponse.ts

```diff
@@ -28,6 +28,16 @@ type MockStableValueConfig = {
   count?: number;
 };
```

Comment:
Mostly this file was moved around. The changes (which look a little weird in the diff) are refactoring a small amount of duplication across two of the functions.